sw/device/silicon_creator/rom_exts/docs/manifest.md

```diff
@@ -8,457 +8,87 @@ aliases:
-*   RSA Signing and Verification is done with 3072-bit keys.
-*   The `Hash` operation may be done with SHA2-256, SHA3-256, SHA3-384, or
-    SHA3-512, depending on the Signature Algorithm Identifier provided in the
-    manifest. The `Hash` operation is required to provide a security level
-    equivalent to that provided by the signature.
+*   `modulus`:  Modulus of the signer's 3072-bit RSA public key.
 
-## Signing
+*   `exponent`: Exponent of the signer's RSA public key. The only values
+    supported by OpenTitan are 3 and 65537.
 
-This is usually done on a host machine, which does not have access to a real OT
-device. Thus there has to be a way for the host signer to predict the value of
-`device_usage_value` and `system_state_value`, as they would be calculated
-on-device.
+*   `identifier`: Image identifier used to identify boot stage images. The
+    value of this field must be `0x4552544f` (ASCII: "OTRE") for a `ROM_EXT`
+    image and `0x3042544f` (ASCII: "OTB0") for the first owner stage.
 
-The `private_key` is never available on-device, in this scheme.
+*   `length`: Length of the image (including the manifest) in bytes.
 
-```
-system_state_value := predict_system_state_value()
-device_usage_value := predict_device_usage_value(rom_ext.usage_constraint)
+*   `version_major`: Major version of the image.
 
-message := system_state_value || device_usage_value || signed_area(rom_ext)
-signature := RSASSA-PKCS1-V1_5-SIGN(private_key, message)
-```
+*   `version_minor`: Minor version of the image.
 
-## Validation
+*   `security_version`: Security version of the image used for anti-rollback
+    protection.
 
-Usually, validation is done on-device, where `device_usage_value` and
-`system_state_value` can be calculated.
+*   `timestamp`: Unix timestamp that gives the creation time of the image,
+    seconds since 00:00:00 on January 1, 1970 UTC (the Unix Epoch).
 
-However, we also want to be able to validate the signature off-device, so still
-need a way to predict the value of `device_usage_value` and
-`system_state_value`. Off-device, we will use the same `predict_` systems as
-used above.
+*   `binding_value`: Binding value used by the [key manager][key_manager] to
+    derive secret values. A change in this value changes the secret value of
+    the key manager, and consequently, the versioned keys and identity seeds
+    generated at subsequent boot stages.
 
-This document does not cover how a device should ensure that `public_key` is
-allowed to validate images for a given device.
+*   `max_key_version`: Maximum allowed version for keys generated by the
+    [key manager][key_manager] at the next boot stage.
 
-```
-system_state_value := calculate_system_state_value()
-device_usage_value := calculate_device_usage_value(rom_ext.usage_constraint)
-signature := rom_ext.signature
+*   `code_start`: Offset of the start of the executable region of the image
+    from the start of the manifest in bytes. Must be word aligned.
```

Comment:
Lets explicitly state `32-bit word` or `4-byte word`. Word can be a bit ambiguous depending on the context.